src/build/languages.js

```diff
@@ -1,5 +1,4 @@
-module.exports = {
+exports.languages = {
```

Comment:
To be honest, I still don't see the point. As far as I know this is a matter of preference and its use will depend greatly on the way we're going to compose our module. Being this just the only export, I don't see this syntactic change as if it's bringing some benefit to the table.

The examples you reference are composed like that (in the case of `supportedExtensions` and `supportedFolders`) because when I first built the extension I thought that they were going to export more items. Finally they remained like that. No other reason behind it. In fact, I've thought of refactoring these modules for a long time but I've been a little bit careless with style in this project.

The way I see it, if I know my module is going to only export one item I don't like creating extra properties. It's like a `default` for ES5, in this case.

You see I may have also something in my inheritance that leans me towars `bizantine discussions` 😜 

In order to close this, let's keep it this way for the sake of consistency as I myself haven't made it clear in the code.